javascript/javascript1/week4/js1w4-hw.js

```diff
@@ -0,0 +1,137 @@
+// VOICE ASSISTANT COMMAND HOME WORK
+
+let string = "Hello my name is Benjamin";
```

Comment:
never name variables `string`! This is a reserved keyword that represents the `string` type